tests/test_admin_logs.py

```
@@ -0,0 +1,117 @@
+    return s
+
+
+def _write_lines(path: str, lines):
+    with open(path, "w", encoding="utf-8") as fh:
+        for l in lines:
+            fh.write(l.rstrip("\n") + "\n")
+
+
+def _write_gz(path: str, lines):
+    with gzip.open(path, "wt", encoding="utf-8") as fh:
+        for l in lines:
```

Ruff (0.13.1)

Ambiguous variable name: `l`

(E741)